### ISSUE TITLE
Unexpected token =

### ISSUE THREAD

**Issue by enesozturk** [software-mansion/react-native-reanimated#1379]
## Description

Hi! I have a project that I have been using **react-native-reanimated@1.13.1**. I tried to update reanimated to v2. I followed all the steps in [installation docs](https://docs.swmansion.com/react-native-reanimated/docs/next/installation). After finish the setup I got the error below. 

In addition to this. After a while, I moved the project to expo. After that, tried upgrading to reanimated v2 again. Followed the [expo's docs](https://docs.expo.io/versions/latest/sdk/reanimated/). I got the same error here.

I am not sure what causes to this. Maybe any package that I have in my project. Do you have any idea? Thanks in advance.

### Screenshots

![Simulator Screen Shot - iPhone 11 - 2020-10-25 at 21 33 44](https://user-images.githubusercontent.com/19428358/97115788-c9a35080-1709-11eb-95ef-24058f523be2.png)

### Expected behavior

Working with Reanimated v2

## Package versions

```json
"dependencies": {
    "@expo/vector-icons": "^10.2.1",
    "@react-native-community/masked-view": "^0.1.10",
    "@react-navigation/bottom-tabs": "^5.9.1",
    "@react-navigation/native": "^5.7.5",
    "@react-navigation/stack": "^5.9.2",
    "@types/react-native-vector-icons": "^6.4.6",
    "@types/react-redux": "^7.1.9",
    "expo": "~39.0.2",
    "expo-application": "~2.3.0",
    "expo-font": "^8.3.0",
    "expo-status-bar": "~1.0.2",
    "react": "16.13.1",
    "react-dom": "16.13.1",
    "react-native": "https://github.com/expo/react-native/archive/sdk-39.0.2.tar.gz",
    "react-native-gesture-handler": "~1.7.0",
    "react-native-reanimated": "^1.13.1",
    "react-native-safe-area-context": "3.1.4",
    "react-native-screens": "~2.10.1",
    "react-native-svg": "^12.1.0",
    "react-native-vector-icons": "^7.1.0",
    "react-native-web": "~0.13.12",
    "react-redux": "^7.2.1",
    "reanimated-bottom-sheet": "^1.0.0-alpha.22",
    "redux": "^4.0.5",
    "redux-thunk": "^2.3.0"
  },
  "devDependencies": {
    "@types/react": "~16.9.35",
    "@types/react-dom": "~16.9.8",
    "@types/react-native": "~0.63.2",
    "react-native-web": "~0.13.7",
    "typescript": "~3.9.5"
  },
```

- React: 16.13.1
- React Native: https://github.com/expo/react-native/archive/sdk-39.0.2.tar.gz
- React Native Reanimated: ^2.0.0-alpha.7
- NodeJS: 10.17.0

**Comment by terrysahaidak** [comment]
Quick fix here is to upgrade your node version to 12.x

This will be deployed bwith the next version.

**Comment by enesozturk** [comment]
Looks like it is solved after upgrading Node JS to v12.19.0.
Thanks!
